third_party/nvidia/hopper/lib/Transforms/WarpSpecialization/WSLowerMem.cpp

```diff
@@ -134,25 +131,23 @@ createLocalCopy(const DenseMap<Channel *, Value> &bufferMap, Channel *channel,
   OpBuilderWithAsyncTaskIds builder(dstOp);
   builder.setAsyncTaskIdsFromOp(dstOp);
   builder.setInsertionPoint(dstOp);
+  builder.setLoopScheduleInfo(dstOp);
```

Comment:
This call is unnecessary because the `builder` will set the information. However, for now I added it for clarity since I expect future changes to mirror existing behavior.